src/panels/lovelace/entity-rows/hui-lock-entity-row.ts

```diff
@@ -56,15 +48,13 @@ class HuiLockEntityRow extends LitElement implements EntityRow {
     `;
   }
 
-  protected renderStyle(): TemplateResult {
-    return html`
-      <style>
-        mwc-button {
-          color: var(--primary-color);
-          font-weight: 500;
-          margin-right: -0.57em;
-        }
-      </style>
+  static get styles(): CSSResult {
+    return css`
+      mwc-button {
+        color: var(--primary-color);
```

Comment:
color and font-weight are actually no longer needed, as that's the default. It's left from the conversion from paper-button to mwc-button.